### PR TITLE
Migrate basic warehouse 

### PR DESCRIPTION
**Pull request by luoyuctl**
What type of PR is this?
Migrate basic warehouse

Add one of the following kinds:
/kind feature

What this PR does / why we need it:
For the subsequent migration of other modules

Specified Reivewers:
/assgin @liuhaoyang

Does this PR introduce a user-facing change?
NONE

### CODE REVIEW

File: pkg/docker/client.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2021 Terminus, Inc.
+
```
Comment (liuhaoyang) [inline pkg/docker/client.go:2]: The `docker-client` provider should be provided by erda-infra

File: pkg/stream/writer/buffer.go [hunk 2]
```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2021 Terminus, Inc.
```
Comment (liuhaoyang) [inline pkg/stream/writer/buffer.go:1]: The buffer pkg already exists in erda-infra

**Review by liuhaoyang — CHANGES_REQUESTED** [review]

**Review by liuhaoyang — CHANGES_REQUESTED** [review]
Why are the common functions of file and json scattered in different packages such as `pkg/utils` and `module/monitor/utils` ?

**Comment by luoyuctl** [comment]
> Why are the common functions of file and json scattered in different packages such as `pkg/utils` and `module/monitor/utils` ?
pkg/utils are allready deleted now

File: pkg/docker/client.go (hunk 1 above)
Comment (luoyuctl) [reply pkg/docker/client.go]: done

**Comment by luoyuctl** [comment]
> Why are the common functions of file and json scattered in different packages such as `pkg/utils` and `module/monitor/utils` ?

**Comment by luoyuctl** [comment]
> Why are the common functions of file and json scattered in different packages such as `pkg/utils` and `module/monitor/utils` ?
Duplicate of #

**Review by liuhaoyang — APPROVED** [review]

**Pull request merged**